manifest.json

```diff
@@ -2310,6 +2310,101 @@
                     ]
                 }
             }
-        }
+        },
+		"eia485|Banane9.BetterLogixWiresThatCanScroll": {
```

Comment:
I'd just use `Banane9.BetterLogixWiresThatCanScroll` for the GUID, as you've used that as a namespace for the other GUID's as well. The authors section still contains eia485's involvement.